lib/hanami/events/formatter/plain_text.rb

```diff
@@ -0,0 +1,21 @@
+module Hanami
+  module Events
+    class Formatter
+      class PlainText
+        def initialize(data)
+          @data = data
+        end
+
+        def format
+          "Events:\n#{formatted_events}"
+        end
+
+        private
+
+        def formatted_events
+          @data.map { |d| "\t#{d[:title].inspect}" }.join("\n")
```

Comment:
we use `title` in this case, but previously we used `event_name`. Should we stick to `event_name` to be more consistent?